src/app/teams/teams-view.component.ts

```diff
@@ -204,7 +204,7 @@ export class TeamsViewComponent implements OnInit, AfterViewChecked, OnDestroy {
       const docsWithName = docs.map(mem => ({ ...mem, name: mem.userId && mem.userId.split(':')[1], avatar: src(mem) }));
       this.leader = docsWithName.find(mem => mem.isLeader) || { userId: this.team.createdBy, userPlanetCode: this.team.teamPlanetCode };
       this.members = docsWithName.filter(mem => mem.docType === 'membership')
-        .sort((a, b) => memberCompare(a, this.leader) ? -1 : 0);
+        .sort((a, b) => memberCompare(a, this.leader) ? -1 : 0).sort(compareFunc);
```

Comment:
You should add the new logic into the original `sort` instead of adding a new `sort` function.